packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -234,6 +234,27 @@ QUnit.module('options changing', {
         assert.equal(this.input.outerHeight(), this.element.height(), 'input outer height should be equal widget height');
         assert.equal(this.input.outerWidth(), this.element.width(), 'input outer width should be equal widget width');
     });
+
```

Comment:
Implementation of isDirty in editor(rangeSlider, DateRange) but tests written in text box and form. I think that tests should be written for the corresponding components and test not in final integration with form.